test/Microsoft.AspNetCore.Mvc.Core.Test/Formatters/StringOutputFormatterTests.cs

```diff
@@ -53,13 +62,17 @@ public void CanWriteResult_DefaultContentType()
         {
             // Arrange
             var formatter = new StringOutputFormatter();
-
             var context = new OutputFormatterWriteContext(
                 new DefaultHttpContext(),
                 new TestHttpResponseStreamWriterFactory().CreateWriter,
                 typeof(string),
                 "Thisisastring");
 
+            // For example, this can happen when a request is received without any Accept header OR a request
+            // is from a browser (in which case this ContentType is set to null by the infrastructure when
+            // RespectBrowserAcceptHeader is set to false)
+            context.ContentType = new StringSegment();
```

Comment:
What is the difference between leaving `ContentType` alone and setting it to an empty `StringSegment`? If you think this case is important, add a test but leave this one alone.